tooling/markdown.rst

```diff
@@ -0,0 +1,58 @@
+.. _markdown:
+
+++++++++
+Markdown
+++++++++
+
+Thanks to Peter Kriens, Alloy supports Markdown syntax. Files written in Alloy Markdown have an ``.md`` extension, just like regular Markdown files. The Alloy Analyzer can read in Markdown files the same way it reads in ``.als`` files.
+
+For an example of an Alloy Markdown file, see Peter Kriens's `Dining Philosophers`_.
+
+.. _Dining Philosophers: https://github.com/pkriens/pkriens.github.io/blob/master/philosophers.md
+
+Header
+~~~~~~
```

Comment:
Could you make these `==` instead of `~~` for consistency with the other pages?

Reply:
Fixed in 3ac3b2d